lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -348,6 +341,11 @@ def model_create_prm_any_baseline_building(user_model, building_type, climate_zo
         return false
       end
 
+      # Apply the minimum damper positions, assuming no DDC control of VAV terminals
+      model.getAirLoopHVACs.sort.each do |air_loop|
+        air_loop_hvac_apply_minimum_vav_damper_positions(air_loop, false)
+      end
+
```

Comment:
Move the loop so the methods is called after the sizing run with the final baseline HVAC system is available so the maximum air flow for terminals can be retrieved.